### PR TITLE
feat: add graceful support for max Celo votes

### PR DESCRIPTION
**Pull request by andyhass**
### 📝 Description

There are a maximum number of Celo validator groups that can be voted for.  Previously, this error was not handled that well.  The user would be able to select the option to vote, but would then see this error message when choosing an amount to vote:

Screen Shot 2022-08-04 at 12 28 33 PM

With this change, we indicate that the user cannot vote in staking management dashboard and prevent them from getting to the voting flow:

Screen Shot 2022-08-05 at 10 33 50 AM

### ❓ Context

- **Impacted projects**: `LLD, LLC` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

See above

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Comment by aelmilligy-ledger** [comment]
Hey @andyhass,
Thank you for your contribution. Some things to add:
- Could you add changeset (pnpm changeset)?
- Have a look on the conflicting files
Best regards.

**Comment by andyhass** [comment]
> Hey @andyhass, Thank you for your contribution. Some things to add:
> 
> * Could you add changeset (pnpm changeset)?
> * Have a look on the conflicting files
>   Best regards.

Hey @aelmilligy-ledger - thanks for reviewing.  I've added the changelog and fixed up the merge conflicts as requested 😄

File: apps/ledger-live-desktop/src/renderer/families/celo/ManageModal/ManageModal.jsx [hunk 1]
```diff
@@ -45,9 +46,10 @@ const ManageModal = ({ name, account, ...rest }: Props) => {
     },
     [dispatch, account],
   );
-
+  const groupsVotedFor = [...new Set(celoResources.votes.map(v => v.validatorGroup))];
```
Comment (haammar-ledger) [inline apps/ledger-live-desktop/src/renderer/families/celo/ManageModal/ManageModal.jsx:49]: Maybe `useMemo` here?

Comment (haammar-ledger) [inline apps/ledger-live-desktop/src/renderer/families/celo/ManageModal/ManageModal.jsx:49]: Maybe `useMemo` would be useful here?...

**Review by haammar-ledger — APPROVED** [review]

**Pull request merged**